actions/recipe_test.go

```diff
@@ -314,7 +314,8 @@ actions:
 }
 
 func runTestWithSubRecipes(t *testing.T, test testSubRecipe, templateVars ...map[string]string) actions.Recipe {
-	var context debos.DebosContext
+	var common debos.CommonContext
+	var context = debos.DebosContext { &common, "", "" }
```

Comment:
Unless you want to use common this can be `context := debos.DebosContext { &debos.CommonContext{}, "", ""}`